sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreConf.scala

```diff
@@ -27,6 +27,12 @@ class StateStoreConf(
 
   def this() = this(new SQLConf)
 
+  /**
+   * Size of MaintenanceThreadPool to perform maintenance tasks
+   * for StateStore
+   */
+  val maintenanceThreadPoolSizeForStateStore: Int = sqlConf.stateStoreMaintenanceThreads
```

Comment:
If we are going to have different names here, it may probably mean either is not clear.